docs/reference/commandline/service_create.md

```diff
@@ -424,7 +424,7 @@ The following options can only be used for named volumes (`type=volume`);
       Options specific to a given volume driver, which will be passed to the
       driver when creating the volume. Options are provided as a comma-separated
       list of key/value pairs, for example,
-      <tt>volume-opt=some-option=some-value,some-other-option=some-other-value</tt>.
+      <tt>volume-opt=some-option=some-value,volume-opt=some-other-option=some-other-value</tt>.
```

Comment:
Is this still not quite correct? I think the other options like this, it is like `--volume-opt key=value --volume-opt otherkey=othervalue`